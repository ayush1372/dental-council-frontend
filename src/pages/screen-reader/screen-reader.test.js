import '@testing-library/jest-dom';

import { render, screen } from '@testing-library/react';

import { ScreenReader } from './index';

test('Screen Reader component snapshot', () => {
  const { container } = render(<ScreenReader />);
  expect(container).toMatchSnapshot();
});

test('Information related to the various screen readers', () => {
  render(<ScreenReader />);
  const TextElement = screen.getByTestId('infoId');
  expect(TextElement).toBeInTheDocument();
});
