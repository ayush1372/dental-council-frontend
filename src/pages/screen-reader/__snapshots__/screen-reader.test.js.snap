// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen Reader component snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1m8jj57-MuiContainer-root"
  >
    <h2
      class="MuiTypography-root MuiTypography-h2 css-8ssjj2-MuiTypography-root"
      data-testid="infoId"
    >
      Information related to the various screen readers
    </h2>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9syv9d-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorPrimary MuiChip-filledPrimary cardType css-qbiv9q-MuiChip-root"
          >
            <span
              class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
            >
              Free
            </span>
          </div>
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <div
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-dpt4p7-MuiTypography-root"
              data-testid="screenTitle"
            >
              Non Visual Desktop Access (NVDA)
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <span
                class="MuiTypography-root MuiTypography-body3 MuiTypography-gutterBottom css-1lx8z1t-MuiTypography-root"
              >
                Website:
              </span>
              <a
                class="MuiTypography-root MuiTypography-body3 MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                data-testid="screenWebsite"
                href="https://www.nvaccess.org/"
                target="_blank"
              >
                NVDA
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9syv9d-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorPrimary MuiChip-filledPrimary cardType css-qbiv9q-MuiChip-root"
          >
            <span
              class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
            >
              Free
            </span>
          </div>
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <div
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-dpt4p7-MuiTypography-root"
              data-testid="screenTitle"
            >
              System Access To Go
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <span
                class="MuiTypography-root MuiTypography-body3 MuiTypography-gutterBottom css-1lx8z1t-MuiTypography-root"
              >
                Website:
              </span>
              <a
                class="MuiTypography-root MuiTypography-body3 MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                data-testid="screenWebsite"
                href="https://www.satogo.com/en/"
                target="_blank"
              >
                Satogo
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9syv9d-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorPrimary MuiChip-filledPrimary cardType css-qbiv9q-MuiChip-root"
          >
            <span
              class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
            >
              Commercial
            </span>
          </div>
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <div
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-dpt4p7-MuiTypography-root"
              data-testid="screenTitle"
            >
              Hal
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <span
                class="MuiTypography-root MuiTypography-body3 MuiTypography-gutterBottom css-1lx8z1t-MuiTypography-root"
              >
                Website:
              </span>
              <a
                class="MuiTypography-root MuiTypography-body3 MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                data-testid="screenWebsite"
                href="https://yourdolphin.com/ScreenReader"
                target="_blank"
              >
                Dolphin Screen Reader
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9syv9d-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorPrimary MuiChip-filledPrimary cardType css-qbiv9q-MuiChip-root"
          >
            <span
              class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
            >
              Commercial
            </span>
          </div>
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <div
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-dpt4p7-MuiTypography-root"
              data-testid="screenTitle"
            >
              Job Access With Speech (JAWS)
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <span
                class="MuiTypography-root MuiTypography-body3 MuiTypography-gutterBottom css-1lx8z1t-MuiTypography-root"
              >
                Website:
              </span>
              <a
                class="MuiTypography-root MuiTypography-body3 MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                data-testid="screenWebsite"
                href="https://www.freedomscientific.com/products/software/jaws/"
                target="_blank"
              >
                JAWS
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 css-11l5t4l-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9syv9d-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorPrimary MuiChip-filledPrimary cardType css-qbiv9q-MuiChip-root"
          >
            <span
              class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
            >
              Commercial
            </span>
          </div>
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <div
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-dpt4p7-MuiTypography-root"
              data-testid="screenTitle"
            >
              SuperNova Magnifier & Screen Reader
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <span
                class="MuiTypography-root MuiTypography-body3 MuiTypography-gutterBottom css-1lx8z1t-MuiTypography-root"
              >
                Website:
              </span>
              <a
                class="MuiTypography-root MuiTypography-body3 MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                data-testid="screenWebsite"
                href="https://yourdolphin.com/SuperNova"
                target="_blank"
              >
                SuperNova
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
